prettier.py

```diff
@@ -58,6 +59,13 @@ def prettify_code(edit, view, region):
 # Run prettier an a file
 class PrettierCommand(sublime_plugin.TextCommand):
     def run(self, edit):
+        extensions = settings.get('extensions')
+
+        # Check if the file extension is allowed
+        file_extension = os.path.splitext(self.view.file_name())[1][1:]
+        if extensions and not file_extension in extensions:
```

Comment:
It's clearer to write this as:

```python
if extensions and file_extension not in extensions:
```

It'd also be nice to log out why it's not working, e.g 

```python
print('Prettier can\'t format .%s files' % file_extension)
```